CHANGES.md

```diff
@@ -1,5 +1,11 @@
 # CHANGELOG
 
+## 1.19.4
+
+### :sparkles: New features
+
+- Improve selected colors [Taiga #5805](https://tree.taiga.io/project/penpot/issue/5805)
```

Comment:
I think you should use the US link here: https://tree.taiga.io/project/penpot/us/5805

Reply:
Fixed!, thanks!